app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -115,4 +122,76 @@ export class ContractsController {
 
     return result;
   }
+
+  _renderSummary(remoteCsvData){
+    this._renderTendersMetricsBox(remoteCsvData.tendersData);
+    this._renderContractsMetricsBox(remoteCsvData.contractsData);
+  }
+
+  _renderTendersMetricsBox(tendersData){
+    // Calculations
+    const amountsArray = tendersData.map((tender) => parseFloat(tender.initial_amount) );
```

Comment:
The point of destructuring if quite useful in this sort of situations:
```suggestion
    const amountsArray = tendersData.map(({ initial_amount = 0 }) => parseFloat(initial_amount) );
```

Unless you know (because you set it up before) that a variable can't come `undefined`, this approach is better to handle data that we don't know (i.e. comes from DB). If `tender` comes null or undefined, `parseFloat` won't never fail

Reply:
This is great, thanks!